src/http/content.cr

```diff
@@ -3,16 +3,49 @@ require "http/common"
 module HTTP
   # :nodoc:
   module Content
+    CONTINUE = "HTTP/1.1 100 Continue\r\n\r\n"
+
+    @continue_sent = false
```

Comment:
Wouldn't be good to expose it to the outside world?
I'm thinking `getter? continue_sent : Bool = false`.

Reply:
For what purpose? The entire types are all `nodoc`.

Reply:
That means internal. And unless a getter would actually be used anywhere in stdlib, it shouldn't be necessary to add it.

Reply:
ok, so `expects_continue` should be better defined with `setter?` instead of `property?` then.